tests/ops/op_math/test_control.py

```diff
@@ -74,59 +72,9 @@ def my_op(a, b, c):
         qml.CRY(-0.123, wires=[5, 3]),
         qml.CRX(-0.789, wires=[5, 2]),
     ]
-    for tape in [tape1.expand(depth=2), tape2.expand(depth=1)]:
-        assert_equal_operations(tape.circuit, expected)
-
-
-class TestAdjointOutsideQueuing:

```

Comment:
This sort of thing is tested in the unittests for `Controlled`.  

Think of this block as tests for `ControlledOperation` rather than tests for `qml.ctrl`.